pkg/migrations/op_add_column.go

```diff
@@ -177,12 +175,7 @@ func (o *OpAddColumn) Complete(ctx context.Context, l Logger, conn db.DB, s *sch
 	}
 
 	if o.Column.Unique {
-		_, err = conn.ExecContext(ctx, fmt.Sprintf("ALTER TABLE %s ADD CONSTRAINT %s_%s_key UNIQUE USING INDEX %s",
-			pq.QuoteIdentifier(o.Table),
-			o.Table,
-			o.Column.Name,
-			UniqueIndexName(o.Column.Name)))
-		if err != nil {
+		if err := NewAddConstraintUsingUniqueIndex(conn, o.Table, o.Column.Name, UniqueIndexName(o.Column.Name)).Execute(ctx); err != nil {
```

Comment:
nit: for a line this long I think it's better to do the assign to `err` on one line and the `if` check on the next.